src/snapred/backend/data/LocalDataService.py

```diff
@@ -236,11 +240,11 @@ def _findMatchingDirList(self, pattern, throws=True) -> List[str]:
 
     def _constructCalibrationStateRootPath(self, stateId):
         # TODO: Propagate pathlib through codebase
-        return f"{self.instrumentConfig.calibrationDirectory}/Powder/{str(stateId)}/"
+        return f"{self.instrumentConfig.calibrationDirectory}/{str(stateId)}/diffraction/"
```

Comment:
this points to `Config["instrument.calibration.home"]` (` /SNS/SNAP/shared/Calibration`)
so wouldnt this be incorrect now?

Reply:
I just had to fix that so it would work on analysis.  The "Powder" seemed to be added twice.  Can you clear up the confusion?  Is the `InstrumentConfig` json coming from SNS incorrect?

Reply:
it gets set [here](https://github.com/neutrons/SNAPRed/blob/01c5f148e9ac77b86b716a423c11c9575ae003d3/src/snapred/backend/data/LocalDataService.py#L103)

Reply:
Just let me know what the final path should look like and I'll make sure its working like that.  **I'm also wondering why `Config["instrument.calibration.home"]` isn't referenced directly here.**

Reply:
There is a conflict as to which params should go into the one stored on sns and the config that should be shipped with the software.  The final say on that was delayed till after phase 3 though calibration directory will probably stay in the `application.yml`

Reply:
OR, is it `Config["instrument.calibration.powder.home"]`?

Reply:
> Just let me know what the final path should look like and I'll make sure its working like that. I'm also wondering why Config["instrument.calibration.home"] isn't referenced directly here?
The short answer is rust, it was something added early.

Reply:
```suggestion
        return f"{Config['instrument.calibration.powder.home']}/{str(stateId)}/diffraction/"
```